### DISCUSSION TITLE
qemu: uncaught target signal 11 (Segmentation fault) - core dumped

### DISCUSSION THREAD

**Discussion by iamanuradha** [docker-library/rabbitmq#612] [General]
On Mac M1, I keep on getting this error. Tried different versions and supported architecture and platform options provided. Searching for the error came across https://github.com/docker/for-mac/issues/6204 stating "Please encourage the author of this container to supply an arm64 or multi-arch image, not just an Intel one. Now that M1 is a mainstream platform, we think that most container authors will be keen to do this."
Blocked on this and want o know if any work around of fix available for this issue

**Comment by Zerpet** [comment]
This image provides multi-arch support. See "Supported architectures" here:

https://hub.docker.com/_/rabbitmq

This just works fine in my M1

```
docker run --platform linux/arm64 --rm rabbitmq:3
```

**Comment by michaelklishin** [comment]
Duplicate of #593?

**Comment by lukebakken** [comment]
@iamanuradha I converted this to a discussion because there is not yet evidence of a bug that we can address. @Zerpet confirmed that this image works on his M1-based computer.

Please provide a transcript showing all commands you run as well as their output. Save it to a file and attach it to your response.
